about/about-kubernetes-services.md

```diff
@@ -0,0 +1,147 @@
+---
+title: About Kubernetes Services
+description: Learn about Kubernetes services!
+---
+
+> <span class="glyphicon glyphicon-info-sign"></span> This guide provides optional background education, including
+> education that is not specific to {{site.prodname}}.
+{: .alert .alert-info}
+
+In this guide you will learn:
+- What are Kubernetes Services?
+- What are the differences between the three main service types and what do you use them for?
+- How do services and network policy interact?
+- Some options for optimizing how services are handled.
+
+### What are Kubernetes Services?
+
+Kubernetes {% include open-new-window.html text='Services'
+url='https://kubernetes.io/docs/concepts/services-networking/service/' %} provide a way of abstracting access to a group
+of pods as a network service. The group of pods backing each service is usually defined using a {% include
+open-new-window.html text='label selector'
+url='https://kubernetes.io/docs/concepts/overview/working-with-objects/labels' %}. 
+
+When a client connects to a Kubernetes service, the connection is load balanced to one of the pods backing the service,
+as illustrated in this conceptual diagram:
+
+![Kubernetes Service conceptual diagram]({{site.baseurl}}/images/k8s-service-concept.svg)
+
+There are three main types of Kubernetes services:
+- Cluster IP - which is the usual way of accessing a service from inside the cluster
+- Node port - which is the most basic way of accessing a service from outside the cluster
+- Load balancer - which uses an external load balancer as a more sophisticated way to access a service from outside the
+  cluster.
+
+### Cluster IP services
+
+The default service type is `ClusterIP`. This allows a service to be accessed within the cluster via a virtual IP
+address, known as the service Cluster IP. The Cluster IP for a service is discoverable through Kubernetes DNS. For
+example, `my-svc.my-namespace.svc.cluster-domain.example`. The DNS name and Cluster IP address remain constant for the
+life time of the service, even though the pods backing the service may be created or destroyed, and the number of pods
+backing the service may change over time.
+
+In a typical Kubernetes deployment, kube-proxy runs on every node and is responsible for intercepting connections to
+Cluster IP addresses and load balancing across the group of pods backing each service. As part of this process
+[DNAT]({{site.baseurl}}/about/about-networking#nat) is used to map the destination IP address from the Cluster IP to the
+chosen backing pod. Response packets on the connection then have the NAT reverse on their way back to the pod which
```

Comment:
... the NAT reversed?...on their way back to the pod that initiated the connection.